app/src/test/java/com/kickstarter/libs/SegmentTest.kt

```diff
@@ -242,6 +243,34 @@ class SegmentTest : KSRobolectricTestCase() {
         this.segmentIdentify.assertValue(user.id())
     }
 
+    @Test
+    fun testSearchCta_Properties() {
```

Comment:
Does this search CTA event also send discover properties? if so, we also need to test for those as well.

Reply:
Actually, it does not. I am refactoring the origin ticket. thanks